cpp/log/ct_extensions.h

```diff
@@ -22,6 +37,8 @@ extern const char kEmbeddedSCTListOID[];
 extern const char kPoisonOID[];
 // Extended Key Usage value for Precertificate signing
 extern const char kPrecertificateSigningOID[];
+// Name constrained intermediate CA may not be logged
+extern const char kNameConstraintNologIntermediateOID[];
```

Comment:
I don't know why this pattern was chosen but as the code interacts with the
OpenSSL C API I decided to follow what existed.

I think we should organize a mini fixit for a couple of days and try to
bring everything up to C++11 etc. Better use of namespaces could be part of
that.

Martin

On 17 August 2015 at 17:23, Pierre Phaneuf <notifications@github.com> wrote:

> In cpp/log/ct_extensions.h
> <https://github.com/google/certificate-transparency/pull/859#discussion_r37206052>
> :
>
> > @@ -22,6 +37,8 @@ extern const char kEmbeddedSCTListOID[];
> >  extern const char kPoisonOID[];
> >  // Extended Key Usage value for Precertificate signing
> >  extern const char kPrecertificateSigningOID[];
> > +// Name constrained intermediate CA may not be logged
> > +extern const char kNameConstraintNologIntermediateOID[];
>
> I suppose you're doing like the others here, but I'm wondering... is there
> a reason to have those visible in the header, rather than ensconced in the
> warmth of an anonymous namespace, in the .cc file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency/pull/859/files#r37206052>
> .
>